resources/assets/themes/dark.scss

```diff
@@ -6,3 +6,8 @@ $secondary: #222;
 $table-striped-bg: rgba(#fff, 0.05);
 
 $es-choices-highlight-color: #000;
+
+/* Hide the clock icon for Chrome */
```

Comment:
It does not hide it but inverts it.
Also the reset button should be changed as weeman mentioned.